packages/ng/demo/app/user/picture/basic/basic.html

```diff
@@ -2,3 +2,5 @@
 <lu-user-picture [user]="anais"></lu-user-picture>
 <p>Without picture</p>
 <lu-user-picture [user]="jeanMich"></lu-user-picture>
+<p>With specified size</p>
+<lu-user-picture [user]="anais" size="50"></lu-user-picture>

```

Comment:
As the final style is apply to `font-size` maybe `size` isn't the best name for this attribute. At first, i thought that `size` correspond to a kind of global ratio for the image

Reply:
With scss, as we do for "override_theme" can we open a kind of variable ? Or maybe it's the time to use css variables :p ? Because, it feels very strange to me to change the style of a component by passing values to attributes

Reply:
I use `font-size` to handle a good scaling of both font/padding and width/height. But it's called `size` because the value you give will actually be the final size in pixel of the picture. 
Regarding theme and variables, the thing as I used it in various interfaces, I realized how much customization is needed size-wise to adapt to each interfaces, therefore, variables wouldn't do the job as you might need various sizes in the same soft.
One solution could be to have mods (mod-small, mod-big, ...)